docs/_sass/_docs-component-list.scss

```
@@ -52,150 +85,347 @@ $grid-transition-delay: 100ms;
     &--img {
 
         &__action-bar {
-            width: 300px;
-            margin-top: 60px;
+            width: 95%;
+
+            @include media("<=tablet") {
+                width: 80%;
+            }
+
+            @include media(">=desktop") {
+                width: 50%;
+            }
+        }
+
+        &__bar {
+            width: 95%;
+
+            @include media("<=tablet") {
+                width: 80%;
+            }
+
+            @include media(">=desktop") {
+                width: 80%;
+            }
+        }
+
+        &__busy-indicator {
+            width: 100%;
+
+            @include media("<=tablet") {
+                width: 40%;
+            }
+
+            @include media(">=desktop") {
+                width: 40%;
+            }
         }
 
         &__message-strip {
-            width: 250px;
-            margin-top: 20px;
+            width: 70%;
+
+            @include media("<=tablet") {
+                width: 40%;
+            }
+
+            @include media(">=desktop") {
+                width: 30%;
+            }
         }
 
         &__status-indicators {
-            width: 250px;
-            margin-top: 60px;
+            width: 70%;
         }
 
 	    &__info-label {
-            width: 250px;
-            margin-top: 60px;
+            width: 70%;
         }
 
         &__button {
-            width: 150px;
-            margin-top: 40px;
+            width: 40%;
+
+            @include media("<=tablet") {
+                width: 30%;
+            }
+
+            @include media(">=desktop") {
+                width: 25%;
```

I'm not sure if this approach on media queries is correct.
In my opinion it should be 
```suggestion
            @include media("<=tablet") {
                &__button {
                    width: 30%;
                }
                
                &__message-strip {
                    ....
                }
            }

            @include media(">=desktop") {
                &__button {
                    width: 25%;
                }
                
                &__message-strip {
                    ....
                }
```

---

It is done like this because of the default values are being applied for tablet<=default<=desktop and not all have a media query. In the end if we have 

`@include media("<=tablet") {
     &__button {
                    width: 30%;
     }
}`

or 
     `&__button {
@include media("<=tablet") {
                    width: 30%;
}`
It is the same thing